com.intellij.diagnostic.PluginException: Cannot create extension (class=nl.hannahsten.texifyidea.startup.StartEvinceInverseSearchListener) [Plugin: nl.rubensten.texifyidea]

### Type of JetBrains IDE (IntelliJ, PyCharm, etc.) and version
IntelliJ IDEA 2022.3.3 (build IU-223.8836.41)

### Operating System 
Windows 11 10.0 (amd64)

### TeXiFy IDEA version
0.7.28

### Description



### Stacktrace
```
com.intellij.diagnostic.PluginException: Cannot create extension (class=nl.hannahsten.texifyidea.startup.StartEvinceInverseSearchListener) [Plugin: nl.rubensten.texifyidea]
	at com.intellij.serviceContainer.ComponentManagerImpl.createError(ComponentManagerImpl.kt:1051)
	at com.intellij.openapi.extensions.impl.XmlExtensionAdapter.doCreateInstance(XmlExtensionAdapter.kt:70)
	at com.intellij.openapi.extensions.impl.XmlExtensionAdapter.createInstance(XmlExtensionAdapter.kt:31)
	at com.intellij.ide.startup.impl.StartupManagerImpl.runPostStartupActivities(StartupManagerImpl.kt:587)
	at com.intellij.ide.startup.impl.StartupManagerImpl.access$runPostStartupActivities(StartupManagerImpl.kt:66)
	at com.intellij.ide.startup.impl.StartupManagerImpl$runStartupActivities$2.invokeSuspend(StartupManagerImpl.kt:184)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:570)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:750)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:677)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:664)
Caused by: com.intellij.diagnostic.PluginException: Cannot load class nl.hannahsten.texifyidea.startup.StartEvinceInverseSearchListener (
  error: com/intellij/openapi/startup/ProjectActivity,
  classLoader=PluginClassLoader(plugin=PluginDescriptor(name=TeXiFy IDEA, id=nl.rubensten.texifyidea, descriptorPath=plugin.xml, path=~\AppData\Roaming\JetBrains\IntelliJIdea2022.3\plugins\TeXiFy-IDEA, version=0.7.28, package=null, isBundled=false), packagePrefix=null, instanceId=219, state=active)
)
	at com.intellij.ide.plugins.cl.PluginClassLoader.loadClassInsideSelf(PluginClassLoader.java:428)
	at com.intellij.ide.plugins.cl.PluginClassLoader.tryLoadingClass(PluginClassLoader.java:242)
	at com.intellij.serviceContainer.ComponentManagerImplKt.doLoadClass(ComponentManagerImpl.kt:1519)
	at com.intellij.serviceContainer.ComponentManagerImplKt.access$doLoadClass(ComponentManagerImpl.kt:1)
	at com.intellij.serviceContainer.ComponentManagerImpl.loadClass(ComponentManagerImpl.kt:941)
	at com.intellij.openapi.extensions.impl.InterfaceExtensionImplementationClassResolver.resolveImplementationClass(InterfaceExtensionImplementationClassResolver.kt:24)
	at com.intellij.openapi.extensions.impl.XmlExtensionAdapter.doCreateInstance(XmlExtensionAdapter.kt:49)
	... 10 more
Caused by: java.lang.NoClassDefFoundError: com/intellij/openapi/startup/ProjectActivity
	at java.base/java.lang.ClassLoader.defineClass2(Native Method)
	at java.base/java.lang.ClassLoader.defineClass(ClassLoader.java:1103)
	at com.intellij.util.lang.UrlClassLoader.consumeClassData(UrlClassLoader.java:264)
	at com.intellij.util.lang.ZipResourceFile.findClass(ZipResourceFile.java:113)
	at com.intellij.util.lang.JarLoader.findClass(JarLoader.java:63)
	at com.intellij.util.lang.ClassPath.findClassInLoader(ClassPath.java:243)
	at com.intellij.util.lang.ClassPath.findClass(ClassPath.java:176)
	at com.intellij.ide.plugins.cl.PluginClassLoader.loadClassInsideSelf(PluginClassLoader.java:420)
	... 16 more
Caused by: java.lang.ClassNotFoundException: com.intellij.openapi.startup.ProjectActivity PluginClassLoader(plugin=PluginDescriptor(name=TeXiFy IDEA, id=nl.rubensten.texifyidea, descriptorPath=plugin.xml, path=~\AppData\Roaming\JetBrains\IntelliJIdea2022.3\plugins\TeXiFy-IDEA, version=0.7.28, package=null, isBundled=false), packagePrefix=null, instanceId=219, state=active)
	at com.intellij.ide.plugins.cl.PluginClassLoader.loadClass(PluginClassLoader.java:217)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:520)
	... 24 more

```

I have the exact same error.

### Type of JetBrains IDE (IntelliJ, PyCharm, etc.) and version
PyCharm 2022.3.3 (build PY-223.8836.43)

### Operating System 
Windows 10 10.0 (amd64)

### TeXiFy IDEA version
0.7.28

### Description



### Stacktrace
```
com.intellij.diagnostic.PluginException: Cannot create extension (class=nl.hannahsten.texifyidea.startup.StartEvinceInverseSearchListener) [Plugin: nl.rubensten.texifyidea]
	at com.intellij.serviceContainer.ComponentManagerImpl.createError(ComponentManagerImpl.kt:1051)
	at com.intellij.openapi.extensions.impl.XmlExtensionAdapter.doCreateInstance(XmlExtensionAdapter.kt:70)
	at com.intellij.openapi.extensions.impl.XmlExtensionAdapter.createInstance(XmlExtensionAdapter.kt:31)
	at com.intellij.ide.startup.impl.StartupManagerImpl.runPostStartupActivities(StartupManagerImpl.kt:587)
	at com.intellij.ide.startup.impl.StartupManagerImpl.access$runPostStartupActivities(StartupManagerImpl.kt:66)
	at com.intellij.ide.startup.impl.StartupManagerImpl$runStartupActivities$2.invokeSuspend(StartupManagerImpl.kt:184)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:570)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:750)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:677)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:664)
Caused by: com.intellij.diagnostic.PluginException: Cannot load class nl.hannahsten.texifyidea.startup.StartEvinceInverseSearchListener (
  error: com/intellij/openapi/startup/ProjectActivity,
  classLoader=PluginClassLoader(plugin=PluginDescriptor(name=TeXiFy IDEA, id=nl.rubensten.texifyidea, descriptorPath=plugin.xml, path=~\AppData\Local\JetBrains\Toolbox\apps\PyCharm-P\ch-0\223.8836.43.plugins\TeXiFy-IDEA, version=0.7.28, package=null, isBundled=false), packagePrefix=null, instanceId=84, state=active)
)
	at com.intellij.ide.plugins.cl.PluginClassLoader.loadClassInsideSelf(PluginClassLoader.java:428)
	at com.intellij.ide.plugins.cl.PluginClassLoader.tryLoadingClass(PluginClassLoader.java:242)
	at com.intellij.serviceContainer.ComponentManagerImplKt.doLoadClass(ComponentManagerImpl.kt:1519)
	at com.intellij.serviceContainer.ComponentManagerImplKt.access$doLoadClass(ComponentManagerImpl.kt:1)
	at com.intellij.serviceContainer.ComponentManagerImpl.loadClass(ComponentManagerImpl.kt:941)
	at com.intellij.openapi.extensions.impl.InterfaceExtensionImplementationClassResolver.resolveImplementationClass(InterfaceExtensionImplementationClassResolver.kt:24)
	at com.intellij.openapi.extensions.impl.XmlExtensionAdapter.doCreateInstance(XmlExtensionAdapter.kt:49)
	... 10 more
Caused by: java.lang.NoClassDefFoundError: com/intellij/openapi/startup/ProjectActivity
	at java.base/java.lang.ClassLoader.defineClass2(Native Method)
	at java.base/java.lang.ClassLoader.defineClass(ClassLoader.java:1103)
	at com.intellij.util.lang.UrlClassLoader.consumeClassData(UrlClassLoader.java:264)
	at com.intellij.util.lang.ZipResourceFile.findClass(ZipResourceFile.java:113)
	at com.intellij.util.lang.JarLoader.findClass(JarLoader.java:63)
	at com.intellij.util.lang.ClassPath.findClassInLoader(ClassPath.java:243)
	at com.intellij.util.lang.ClassPath.findClass(ClassPath.java:176)
	at com.intellij.ide.plugins.cl.PluginClassLoader.loadClassInsideSelf(PluginClassLoader.java:420)
	... 16 more
Caused by: java.lang.ClassNotFoundException: com.intellij.openapi.startup.ProjectActivity PluginClassLoader(plugin=PluginDescriptor(name=TeXiFy IDEA, id=nl.rubensten.texifyidea, descriptorPath=plugin.xml, path=~\AppData\Local\JetBrains\Toolbox\apps\PyCharm-P\ch-0\223.8836.43.plugins\TeXiFy-IDEA, version=0.7.28, package=null, isBundled=false), packagePrefix=null, instanceId=84, state=active)
	at com.intellij.ide.plugins.cl.PluginClassLoader.loadClass(PluginClassLoader.java:217)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:520)
	... 24 more

```

--- a/gradle.properties
+++ b/gradle.properties
@@ -4,7 +4,7 @@ pluginVersion = 0.7.28
 #    Note that an xyz branch corresponds to version 20xy.z and a since build of xyz.*
 #    means that the first possible build is the next branch number after xyz, so e.g.
 #    a since build of 173.* is equal to 181
-pluginSinceBuild = 223
+pluginSinceBuild = 231
 
 # Token for releasing to Jetbrains using the Gradle intellij/publishPlugin task, for more information see the Gradle build file.
 intellijPublishToken=mytoken

